src/mnode/mnode-pastel.h

```diff
@@ -618,3 +619,1812 @@ class CTakeDownTicket : public CPastelTicket
 
     void SerializationOp(CDataStream& s, const SERIALIZE_ACTION ser_action) override {}
 };
+
+// Username Change Ticket /////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+	"ticket": {
+		"type": "username",
+		"pastelID": "",    //PastelID of the username
+		"username": "",    //new valid username
+		"fee": "",         // fee to change username
+		"signature": ""
+	},
+ */
+class CChangeUsernameTicket : public CPastelTicket
+{
+public:
+    std::string pastelID;
+    std::string username;
+    CAmount fee{100};
+    std::vector<unsigned char> signature;
```

Comment:
you can use new type v_uint8 (from vector_type.h)

Reply:
fixed